stories/radioButtonGroup.stories.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { boolean, withKnobs } from '@storybook/addon-knobs';
+import notes from './radioButtonGroup.md';
+import { Box, Flex, RadioButtonGroup } from '../src';
+
+const stories = storiesOf('RadioButtonGroup', module);
+stories.addDecorator(withKnobs);
+
+const onChange = value => {
+  console.log('ch ch ch changed', value); // eslint-disable-line no-console
+};
+
+const groupOptions = [
+  {
+    label: 'RadioButton group (a)',
+    value: 'a'
+  },
+  {
+    label: 'RadioButton group (b)',
+    value: 'b'
+  },
+  {
+    label: 'RadioButton group (should be disabled) (c)',
+    value: 'c',
+    disabled: true
+  },
+  {
+    label: 'RadioButton group (d)',
+    value: 'd'
+  }
+];
+
+stories.add(
+  'default',
+  () => (
+    <Box p="larger">
+      <RadioButtonGroup
+        options={groupOptions}
+        name="grouped"
+        onChange={onChange}
+        disabled={boolean('Disabled (Group level)', false)}
+      />
+
+      <Flex flexDirection="row">
```

Comment:
Note that RadioButtonGroup has no opinion on how things render. This is how you would render a group of buttons in a single row.